site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -0,0 +1,137 @@
+
+ * `auth.example.com` - this value would allow users to share the same passkey between `auth.example.com` and `m.auth.example.com`, but it would prevent users from registering or using passkeys on the `login.example.com` legacy domain
+ * `example.com` - this value functions the same as using `auth.example.com` with the added benefit that users would be allowed to register and use passkeys on `login.example.com` _and_ continue using those same passkeys once they are migrated to the new `auth.example.com` domain
+
+==== Workflow settings
+
+image::core-concepts/tenant-configuration-webauthn-workflow.png[Tenant Configuration - WebAuthn workflow settings,width=1200, role=top-cropped,bottom-cropped]
+
+Each WebAuthn workflow has the same set of configuration options.
+
+[.api]
+[field]#Enabled#::
+Enabling this toggle will enable the associated workflow for applications belonging to this tenant. The availability of a workflow can be overridden at the application level using the link:#application-configuration[Application Configuration].
+
+[field]#Authenticator attachment#::
+This setting controls which authenticator attachments are allowed for registration ceremonies when using the associated workflow. See the section link:#authenticator-attachment[below] for recommendations on configuring this option.
+
+[field]#User verification#::
+This setting controls whether user verification is required for registration and authentication ceremonies when using the associated workflow. See the section link:#user-verification[below] for recommendations on configuring this option.
+
+===== Authenticator attachment
+
+There are two link:https://www.w3.org/TR/webauthn-2/#authenticator-attachment-modality[authenticator attachment modalities] in WebAuthn.
+
+ * `platform` - the authenticator is integrated with the client device
+ * `cross-platform` - the authenticator is removable from the client device and can be used with other client devices
+
+The workflow settings allow limiting the associated workflow to one attachment only or allowing either. The value is used to limit eligible authenticators to those that are best suited for the workflow. The authenticator attachment preference only impacts the WebAuthn registration ceremony. These are the recommended values for available WebAuthn workflows:
+
+ * Bootstrap - Cross-platform only. The bootstrap workflow can be used to sign in on a new device using WebAuthn. If the authenticator is integrated with the client device, it cannot be used on another.
+ * Re-authentication - Platform only. In order to ensure that the authenticator is available for repeated logins on the same device, it is best to limit authenticator selection to those integrated with the client device.
```

Comment:
I might argue 'either' is a better value here.

Reply:
Agreed and updated for the bootstrap workflow.

Given that the intent of re-authentication is to streamline repeated logins from the same device, `Platform only` still makes more sense.